sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/LiveVariableAnalysisTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿extern alias csharp;
```

Comment:
We usually put such `extern alias` statements after the license header, at the start of the using directives

Reply:
I've updated that for `ExplodedGraphTests.cs` as well = the source of the copy/paste :)